scripts/controllers/usefulness_controller.ts

```diff
@@ -1,23 +1,23 @@
 import { Controller } from '@hotwired/stimulus'
-
+import { UsefulnessReason } from '../common/enum'
 class UsefulnessController extends Controller {
-    vote: String | null
     launchModal: String | null
-    
+    reasonTargets: any
+    usefulYesTarget: any
+    usefulNoTarget: any
+
     initialize() {
-      this.vote = null
       this.launchModal = ''
     }
   
     static get targets() {
-      return []
+      return ['reason', 'usefulYes', 'usefulNo']
     }
   
     handleVote(event: any) {
       this.launchModal = event.params.launchModal
-      this.vote = this.launchModal === 'useful-yes' ? 'yes' : 'no'
-      const modal = document.querySelector(`#${this.launchModal}`)
-      modal.classList.add('active')
+
+      this[`${this.launchModal}Target`].classList.add('active')
```

Comment:
I hope we're not collapsing property names during minification/bundling 😅😬

Reply:
I've been running webpack in production mode so if it did, I would have seen it. 😄